addons/player-thumb/addon.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Show project thumbnail while loading",
+  "description": "Shows the project's thumbnail instead of its initial state before starting the project and while it loads.",
```

Comment:
The description could probably be better - it can currently be read as
`Shows the project's thumbnail instead of its initial state (before starting the project and while it loads)`
instead of
`Shows the project's thumbnail (instead of its initial state before starting the project) and (while it loads)`

Reply:
```suggestion
  "description": "Shows the project's thumbnail before starting the project and while it loads.",
```

Reply:
```suggestion
  "description": "Shows the project's thumbnail as a placeholder before the project is started and while it loads.",
```